river/metrics/cluster/__init__.py

```diff
@@ -0,0 +1,33 @@
+"""Internal clustering metrics
+
+This submodule includes all internal clustering metrics that are updated with one sample,
+its label and the current cluster centers at a time. Using this, we can track the performance
+of the clustering algorithm without having to store information of all previously passed points.
+
+"""
+
+from .base import InternalClusMetric
```

Comment:
I would simply call this `Metric` now that's in the `cluster` submodule :)

Reply:
I'm just afraid that calling it `Metric` will cause a bit of confusion, since in clustering we still have internal metrics and external metrics. The external metrics are exactly the ones that River has had before, e.g. F1, Cohen, Kappa, Jaccard, etc.

Reply:
Ok but it's the same case for `R2` no? As long as the clustering metrics and the base class are in the `cluster` submodule I think it's fine.

Reply:
You're right. I'm having a short conversation with @jacobmontiel this Thursday, and I think we can finalize on this real quick.

Reply:
This will be renamed as `InternalMetric`

Reply:
Resolved and changed!